README.md

```diff
@@ -63,7 +71,22 @@ For the official list of tools that support FMI, check http://fmi-standard.org/t
 - [ROS fmi_adapter](https://github.com/boschresearch/fmi_adapter) - FMI support for [ROS](http://www.ros.org/), a flexible framework for writing robot software. [APACHE2]
 - [fmi_adapter_ros2](https://github.com/boschresearch/fmi_adapter_ros2) - FMI support for [ROS2](https://index.ros.org/doc/ros2/), the new version of the Robot Operating System. [APACHE2]
 
+## FMI 3
+
+
+FMI 3.0  development is currently in Alpha phase. General information on the FMI 3.0 feature list can be found in https://fmi-standard.org/faq/, and the development is ongoing on the repo https://github.com/modelica/fmi-standard . You can find the latest, automatically generated version of the FMI 3.0 alpha standard at https://fmi-standard.org/docs/3.0-dev/ .
+
+**Warning: FMI 3.0 is still under heavy development, and most of the tools listed in the following they support it only in an experimental way. If you want stable tools, use FMI 2 for the time being.**
+
+Some of the libraries listed in the previous section also have prototypal support for FMI 3 alpha: 
+
+- **Python:** [fmpy](https://github.com/CATIA-Systems/FMPy)  some support for FMI 3 is available in the [`develop`](https://github.com/CATIA-Systems/FMPy/tree/develop) branch.
```

Comment:
fmpy 0.2.19 already contains FMI3.0-alpha.3 support in the release version. Looking at the develop branch is perhaps a good idea to see the latest development.